cheatsheets_draft/Kubernetes_Security_Cheat_Sheet.md

```diff
@@ -270,6 +270,54 @@ spec:
+
+#### Obeservability 
+Service Mesh provides tracing and telemetry metrics that make it easy to understand your system and quickly root cause any problems.
+	
+#### Security
+A service mesh provides security features aimed at securing the services inside your network and quickly identifying any compromising traffic entering your cluster. A service mesh can help you more easily manage security through mTLS, ingress and egress control, and more.
+
+* mTLS and Why it Matters
+		Securing microservices is hard. There are a multitude of tools that address microservices security, but service mesh is the most elegant solution for addressing encryption of on-the-wire traffic within the network.
+		Service mesh provides defense with mutual TLS (mTLS) encryption of the traffic between your services. The mesh can automatically encrypt and decrypt requests and responses, removing that burden from the application developer. It can also improve performance by prioritizing the reuse of existing, persistent connections, reducing the need for the computationally expensive creation of new ones. With service mesh, you can secure traffic over the wire and also make strong identity-based authentication and authorizations for each microservice.
+		We see a lot of value in this for enterprise companies. With a good service mesh, you can see whether mTLS is enabled and working between each of your services and get immediate alerts if security status changes.
+
+* Ingress & Egress Control
+Service mesh adds a layer of security that allows you to monitor and address compromising traffic as it enters the mesh. Istio integrates with Kubernetes as an ingress controller and takes care of load balancing for ingress. This allows you to add a level of security at the perimeter with ingress rules. Egress control allows you to see and manage external services and control how your services interact with them.
+
+#### Operational Control
+A service mesh allows security and platform teams to set the right macro controls to enforce access controls, while allowing developers to make customizations they need to move quickly within these guardrails.
+
+#### RBAC
+A strong Role Based Access Control (RBAC) system is arguably one of the most critical requirements in large engineering organizations, since even the most secure system can be easily circumvented by overprivileged users or employees. Restricting privileged users to least privileges necessary to perform job responsibilities, ensuring access to systems are set to “deny all” by default, and ensuring proper documentation detailing roles and responsibilities are in place is one of the most critical security concerns in the enterprise.
+
+### Implementing Open Policy Agent (OPA) for a centralized policy management 

```

Comment:
any alternatives for it?

Reply:
This topic was one of suggestions and I'm not quite exposed to it. I did some research however couldn't find much resource. Could the team pls help with it? Thanks in advance

Reply:
No worries if you done fair research and you did not find anything that is fine. OPA is kind of the standard now so I am happy that you covered it here.

Reply:
Thanks @mackowski.
 I have done the research to the best of my understanding, would be happy to add details based of the team's guidance.